src/ls/ls.rs

```diff
@@ -196,6 +196,7 @@ fn sort_entries(entries: &mut Vec<PathBuf>, options: &getopts::Matches) {
                     .unwrap_or(std::time::UNIX_EPOCH)
             });
         }
+        entries.reverse(); // Newest first
```

Comment:
Good point; it seems the standard library has a neat struct for reversing sorts, so I used that.